doc/source/contributing.rst

```diff
@@ -333,10 +341,10 @@ Requirements
 ~~~~~~~~~~~~
 
 To build the *pandas* docs there are some extra requirements: you will need to
-have ``sphinx`` and ``ipython`` installed. `numpydoc
+have sphinx and ipython installed. `numpydoc
```

Comment:
You are right, there is very little consistency in the docs about what should and should not be highlighted. My logic was that only code-like text should be highlighted, and the name of a package therefore doesn't qualify. But, you raise a good point about making sure those two package names pop out, so I'll put them back in. Maybe a better rule would be to highlight any conda/pip-installable package name? 

The decision to italicize *pandas* wasn't mine, it was like that in the previous .md file, but I kind of like it: it's like a subtle nod to the fact that *pandas* is the subject of the document.